cdap-proto/src/main/java/co/cask/cdap/proto/profile/ProvisionerInfo.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.profile;
+
+import co.cask.cdap.runtime.spi.provisioner.ProvisionerProperty;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Information about a provisioner name and properties.
+ */
+public class ProvisionerInfo {
+  private final String name;
+  private final String description;
+  private final List<ProvisionerProperty> properties;
```

Comment:
properties should just be a string string map. ProvisionerProperty is like a specification for each property. It is used in the provisioners rest apis, but not in the profile api. Properties here contains the values for each provisioner property.